src/components/AnswersTable/AnswersTable.js

```diff
@@ -0,0 +1,135 @@
+import { html } from '../../shared';
+import styles from './AnswersTable.module.css';
+import check from '../../public/img/check.svg';
+import x from '../../public/img/x.svg';
+
+/**
+ * @typedef {import('../../components/Question').FullAnswerCharacterCategory} FullAnswerCharacterCategory
+ */
+
+/**
+ * @typedef {import('../../components/Question').FullAnswerEpisodeOrLocationCategory} FullAnswerEpisodeOrLocationCategory
+ */
+
+/**
+ * Generates an image element with an icon depending on the correctness of the answer.
+ * @param {'true'|'false'} correct
+ * @returns {HTMLElement}
+ */
+const correctIcon = (correct) => {
+  const img = document.createElement('img');
+  img.className = styles.correctIcon;
+  img.src = correct === 'true' ? check : x;
+  img.alt = '';
+  return img;
+};
+
+/**
+ * Generates a div with character avatars for correct answers and user answers (for the episodes and localization categories)
+ * @param {import('../../components/Question').EpisodeOrLocationAnswers} answers
+ * @param {HTMLDivElement} answersDiv
+ */
+const answerImagesDiv = (answers, answersDiv) => {
+  answers.forEach((character) => {
+    const characterImage = document.createElement('img');
+    characterImage.src = character.image;
+    characterImage.alt = character.name;
+    characterImage.className = styles.answerImage;
+    answersDiv.appendChild(characterImage);
+  });
+};
+
+/**
+ * @param {number} count
+ * @param {FullAnswerCharacterCategory} answer
+ * @returns {[Text, HTMLImageElement, Text, Text, HTMLElement]}
+ */
+const characterQuestionElements = (count, answer) => {
+  const questionNumber = document.createTextNode(`${count}`);
+  const characterImage = document.createElement('img');
+  const correctAnswer = document.createTextNode(`${answer.correctAnswers[0].name}`);
+  const userAnswer = document.createTextNode(`${answer.userAnswers[0] ? answer.userAnswers[0].name : ''}`);
+  const correct = correctIcon(`${answer.correct}`);
```

Comment:
```suggestion
  const correct = correctIcon(answer.correct);
```
Patrz: komentarz do linii 16-22.